### PR TITLE
notebooks 1-4 update

### PR DESCRIPTION
**Pull request by clonker**

### CODE REVIEW

**Review by thempel — APPROVED** [review]

File: notebooks/06-hmm.ipynb [hunk 1]
```diff
@@ -96,9 +97,11 @@
     "axes[0].set_ylabel('$y$')\n",
     "axes[0].set_title('discretization')\n",
     "\n",
-    "lags = range(1, 10)\n",
-    "its = implied_timescales_msm(good_dtraj, lags, 1)\n",
-    "pyemma.plots.plot_implied_timescales(its, ylog=False, ax=axes[1])\n",
+    "models = [MaximumLikelihoodMSM(lagtime=lag).fit_fetch(good_dtraj) for lag in range(1, 11)]\n",
+    "its = implied_timescales(models)\n",
+    "# lags = range(1, 10)\n",
+    "# its = implied_timescales_msm(good_dtraj, lags, 1)\n",
```
Comment (thempel) [inline notebooks/06-hmm.ipynb:103]: remove commented lines?

File: notebooks/06-hmm.ipynb [hunk 2]
```diff
@@ -695,15 +714,17 @@
     "cluster = #FIXME\n",
     "dtrajs = #FIXME\n",
     "\n",
-    "its = implied_timescales_msm(dtrajs, lagtimes=[1, 2, 5, 10, 20, 50], nits=4)\n",
+    "its = implied_timescales(MaximumLikelihoodMSM(lagtime=lag) for lag in [1, 2, 5, 10, 20, 50])\n",
```
Comment (thempel) [inline notebooks/06-hmm.ipynb:717]: Does this really work? 😮

File: notebooks/06-hmm.ipynb [hunk 3]
```diff
@@ -986,9 +1029,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "deeptime",
    "language": "python",
-   "name": "python3"
+   "name": "deeptime"
```
Comment (thempel) [inline notebooks/06-hmm.ipynb:1034]: todo for later: use the same default kernel name.

**Review by thempel — APPROVED** [review]

File: notebooks/06-hmm.ipynb (hunk 2 above)
Comment (clonker) [reply notebooks/06-hmm.ipynb:717]: nope

File: notebooks/06-hmm.ipynb (hunk 1 above)
Comment (clonker) [reply notebooks/06-hmm.ipynb:103]: yup

**Pull request merged**